src/error.rs

```diff
@@ -24,6 +24,8 @@ impl std::error::Error for ColumnFromStrErr {}
 
 impl std::fmt::Display for ColumnFromStrErr {
     fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        use chrono::format;
```

Comment:
This line also seems to be redundant (I am not very sure though).

Reply:
Removed